org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/ui/IGPublisherUI.java

```diff
@@ -0,0 +1,775 @@
+        commandStr.append("- Folder: ").append(folderPath).append("\n");
+        if (!termServer.isEmpty()) commandStr.append("- Terminology Server: ").append(termServer).append("\n");
+        commandStr.append("- No Narrative: ").append(noNarrative).append("\n");
+        commandStr.append("- No Validation: ").append(noValidation).append("\n");
+        commandStr.append("- No Network: ").append(noNetwork).append("\n");
+        commandStr.append("- Track Fragment Usage: ").append(trackFragmentUsage).append("\n");
+        commandStr.append("- Clear Terminology Cache: ").append(clearTermCache).append("\n");
+        commandStr.append("- No Sushi: ").append(noSushi).append("\n");
+        commandStr.append("- Debug: ").append(debug).append("\n");
+        commandStr.append("- Allow Non-conformant Terminology Servers: ").append(allowNonConformantTx).append("\n\n");
+        
+        logTextArea.append(commandStr.toString());
+        
+        // Create a custom output stream that redirects to the TextArea
+        OutputStream customOutputStream = new OutputStream() {
+            private StringBuilder lineBuffer = new StringBuilder();
+            
+            @Override
+            public void write(int b) {
+                char c = (char) b;
+                if (c == '\n') {
+                    // End of line, update the TextArea
+                    final String line = lineBuffer.toString();
+                    SwingUtilities.invokeLater(() -> logTextArea.append(line + "\n"));
+                    lineBuffer = new StringBuilder();
+                } else {
+                    lineBuffer.append(c);
+                }
+            }
+            
+            @Override
+            public void flush() {
+                if (lineBuffer.length() > 0) {
+                    final String line = lineBuffer.toString();
+                    SwingUtilities.invokeLater(() -> logTextArea.append(line));
+                    lineBuffer = new StringBuilder();
+                }
+            }
+        };
+        
+        // Create a PrintStream wrapper for our custom output stream
+        PrintStream customPrintStream = new PrintStream(customOutputStream, true);
+        
+        wasCanceled = false;
+        
+        // Create a Thread object so we can interrupt it if needed
+        Thread publisherThread = new Thread(() -> {
+            // Save the original System.out and System.err
+            PrintStream originalOut = System.out;
+            PrintStream originalErr = System.err;
+            
+            try {
+                // Redirect System.out and System.err to our custom stream
+                System.setOut(customPrintStream);
+                System.setErr(customPrintStream);
+                
+                // Call the Publisher directly
+                Publisher.runDirectly(
+                    folderPath,
+                    termServer.isEmpty() ? null : termServer,
```

Comment:
Setting the terminology server to null here causes this to run without a terminology server, instead of the default server.

I think it should be this:

```java
termServer.isEmpty() ? FhirSettings.getTxFhirProduction() : termServer,
```

Reply:
removed